CoreScriptsRoot/Modules/BackpackScript.lua

```diff
@@ -1608,9 +1626,9 @@ do -- Make the Inventory expand/collapse arrow (unless TopBar)
 			disableGamepadInventoryControl()
 		end
 
-		if InventoryFrame.Visible and GamepadEnabled then
+		if InventoryFrame.Visible then
```

Comment:
why would we bind this if we don't have a gamepad? is this for if you connect one later?